internal/cmdsearch/searchreplace_test.go

```diff
@@ -59,7 +59,7 @@ metadata:
 	},
 	{
 		name: "search replace by value",
```

Comment:
should this file be called putvalue_test.go for consistency?

Reply:
this file contains tests for both search and also put-value, so felt it is better name. Renamed other file to putcomment_test.go as it contains only --put-comment cases.